api/src/test/java/org/pmiops/workbench/api/AppsControllerTest.java

```diff
@@ -84,6 +84,7 @@ public void setUp() {
 
     user = new DbUser();
     createAppRequest = new CreateAppRequest().appType(AppType.RSTUDIO);
+    config.featureFlags.enableRStudioGKEApp = true;
```

Comment:
I'd prefer to remove this and set them explicitly in the tests

Reply:
Good idea, done.